git_machete/cmd.py

```diff
@@ -30,12 +30,15 @@ def __str__(self):
 ORANGE = '\033[00;38;5;208m'
 RED = '\033[91m'
 
+
```

Comment:
Pls bump the version to 2.8.6 since only no externally affecting changes were done, only the internal ones

Reply:
Do you mean, rebase on latest master?

Reply:
Oh sorry, I just meant increasing the version number here in cmd.py, could you pls? In case of any issues it's important to know if people had the original 2.8.5 or the updated version built after your PR is merged

Reply:
Pls also add an entry in RELEASE_NOTES.md

Reply:
Ah, this way, ok, thx!